recipe_scrapers/cookwell.py

```diff
@@ -1,7 +1,29 @@
 from ._abstract import AbstractScraper
+import re
 
 
 class CookWell(AbstractScraper):
     @classmethod
     def host(cls):
         return "cookwell.com"
+
+    def nutrients(self):
+        nutrition = None
+        nutrition_pattern = re.compile(
+            r"{\\\"calories\\\":(?P<calories>\d+),\\\"carbohydrates\\\":(?P<carbohydrates>\d+),\\\"fat\\\":(?P<fat>\d+),\\\"protein\\\":(?P<protein>\d+)}"
```

Comment:
```suggestion
            r'{\\"calories\\":(?P<calories>\d+),\\"carbohydrates\\":(?P<carbohydrates>\d+),\\"fat\\":(?P<fat>\d+),\\"protein\\":(?P<protein>\d+)}'
```

This can be simplified by using `' '` to encapsulate the string removing the need for the additional backslashes.